pkg/rest/config_upload.go

```diff
@@ -173,17 +213,32 @@ func getExistingValuesFromEndpoint(client *http.Client, configType string, url s
 		}
 		values = jsonResp
 	default:
-		var jsonResponse api.ValuesResponse
-		err = json.Unmarshal(resp.Body, &jsonResponse)
-		if util.CheckError(err, "Cannot unmarshal API response for existing objects") {
-			return isDashboard, values, err
+		if theApi.GetPropertyNameOfGetAllResponse() == "" {
```

Comment:
only if it's empty? Your tests explicitly put "values" - shouldn't they put nothing instead? Doesn't matter for the tests, but I tend to look at tests for sample usage